examples/data_sharing.tex

```diff
@@ -0,0 +1,210 @@
+\documentclass[a4paper]{article}
+\usepackage{data}
+
+%%%%% Now we input our variable terms as glossary entries %%%%%
+
+\newglossaryentry{type}{
+	name={Company},
+	first={the ``Company''},
+	text={Company},
+	description={is a commercial undertaking}
+}
+
+\newglossaryentry{undertaking}{
+	name={King of the Eagles},
+	first={King of the Eagles, hereafter `\glsentrytext{type}'},
+	text={the \glsentrytext{type}},
+	description={are Big birds. Live on a hill}
+}
+
+\newglossaryentry{processor}{
+	name={Mirkwood Elves},
+	first={Mirkwood Elves, hereafter `the Processor'},
+	text={Data Processor},
+	description={are the Elves of the Noldor Line}
+}
+
+\newcommand{\services}{Orc-slaying}
+
+\newcommand{\companyAddress}{123 Eirie Road}
+
+
+\newcommand{\processorAddress}{Third Tree, Mirkwood}
+
+\newcommand{\processorEmail}{elves@place.mw}
+
+\title{Data Sharing Agreement between \Glsentrytext{undertaking} \& \Glsentrytext{processor}}
+
+\begin{document}
```

Comment:
> @passamo9: The left alignment on the first line is inconsistent.

^ i don't know which "first line" @passamo9 was referring to tho.

Reply:
I was referring to this first line

<img width="549" alt="image" src="https://user-images.githubusercontent.com/69139574/108000465-555b2880-6fea-11eb-88d0-bd4addc4a13a.png">

Thank you again @yjyao!

Reply:
perhaps set `\setlength{\parindent}{0pt}` at some point? I think you're talking about the indents due to starting paragraphs. Should normal paragraphs be indented? Or should nothing be?

Reply:
I just think that it's off.

The first line *in this instance only* should not be indented, but be in-line with the second line.

Please let me know if you would like for me to clarify

Reply:
I'm sorry, @passamo9, for being a bit thick right now. But could you perhaps circle which exact line you do not want indented, and put that image here? Then we can add a `\noindent` on that line and see if that clears it up.

Reply:
I think this has been fixed by replacing the title page entirely.

Reply:
No worries! This one:

![Image 16-02-2021 at 00 58](https://user-images.githubusercontent.com/69139574/108003165-2648b500-6ff2-11eb-806c-313ed08b71b4.jpg)

Reply:
This doesn't look like it's addressed yet, though this isn't the right line for it. I'm pretty sure it should actually be discussing line 71. I opened a request for change there, so if this conversation could be carried on over there, that'd be great. Also thank you for the clarification! Really helpful)